.github/workflows/build-latest.yaml

```diff
@@ -1,9 +1,9 @@
 ---
-name: Continous integration
+name: Continuous integration
 on:
   push:
-    branches:
-      - master
+    tags:
+      - 'v*'
```

Comment:
This would disabled tests for latest commit in the master, right?

Reply:
Yes, you are correct. Assuming tests on the `master` branch are successful and that releases are tagged from that branch, Docker images will be built. 

If needed, we can make the flow depending on the succes of other flows. This will not protect the repository from broken tags, though.